### PR TITLE
Add Notification Statuses

### PR DESCRIPTION
**Pull request by dhaselhan**
* Add statuses to notifications

### CODE REVIEW

File: services/apps/alcs/src/alcs/notification/notification-submission-status/notification-submission-status.service.ts [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+  }
+
+  private async getSubmission(fileNumber: string) {
+    const submission = await this.notificationSubmissionRepository.findOneBy({
+      fileNumber,
+    });
+
+    if (!submission) {
+      throw new ServiceNotFoundException(
+        `Submission does not exist for provided notice of intent ${fileNumber}. Only notice of intents originated in portal have statuses.`,
```
Comment (mhuseinov) [inline services/apps/alcs/src/alcs/notification/notification-submission-status/notification-submission-status.service.ts:114]: notice of intent -> notification

File: services/apps/alcs/src/providers/typeorm/migrations/1693952313867-add_notification_statuses.ts [hunk 2]
```diff
@@ -0,0 +1,37 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class addNotificationStatuses1693952313867
+  implements MigrationInterface
+{
+  name = 'addNotificationStatuses1693952313867';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      `CREATE TABLE "alcs"."notification_submission_status_type" ("audit_deleted_date_at" TIMESTAMP WITH TIME ZONE, "audit_created_at" TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT now(), "audit_updated_at" TIMESTAMP WITH TIME ZONE DEFAULT now(), "audit_created_by" character varying NOT NULL, "audit_updated_by" character varying, "label" character varying NOT NULL, "code" text NOT NULL, "description" text NOT NULL, "weight" smallint NOT NULL DEFAULT '0', "alcs_background_color" character varying NOT NULL, "alcs_color" character varying NOT NULL, "portal_background_color" character varying NOT NULL, "portal_color" character varying NOT NULL, CONSTRAINT "UQ_7fb8eeb106bb4bb0b0738bd1106" UNIQUE ("description"), CONSTRAINT "PK_9a28eacfd5a7b00f06bdf234bcd" PRIMARY KEY ("code"))`,
```
Comment (mhuseinov) [inline services/apps/alcs/src/providers/typeorm/migrations/1693952313867-add_notification_statuses.ts:10]: looks like table comment is missing

**Review by mhuseinov — APPROVED** [review]

File: services/apps/alcs/src/alcs/notification/notification-submission-status/notification-submission-status.service.ts (hunk 1 above)
Comment (dhaselhan) [reply services/apps/alcs/src/alcs/notification/notification-submission-status/notification-submission-status.service.ts]: Fixed

File: services/apps/alcs/src/providers/typeorm/migrations/1693952313867-add_notification_statuses.ts (hunk 2 above)
Comment (dhaselhan) [reply services/apps/alcs/src/providers/typeorm/migrations/1693952313867-add_notification_statuses.ts:10]: Will add comments when I get to documents.

**Pull request merged**